daemon/indexing/lib/search.js

```diff
@@ -84,7 +84,11 @@ class Listing {
   static async search(query, filters) {
     const esQuery = {
       bool: {
-        must: [],
+        must: [{
+          "match": {
```

Comment:
I think linter will complain about using double quotes. Can you use single quote ?